test/integration/thermal_camera.cc

```diff
@@ -232,7 +258,8 @@ void ThermalCameraTest::ThermalCameraBoxes(
 
 TEST_P(ThermalCameraTest, ThermalCameraBoxes)
 {
-  ThermalCameraBoxes(GetParam());
+  ThermalCameraBoxes(GetParam(), false);
+  ThermalCameraBoxes(GetParam(), true);
```

Comment:
can you separate these two to two individual `TEST_P` with the other called something like `ThermalCameraBoxesHeatSignature`? This is mainly so that it'll be easier to determine what passed / failed by looking the test name when we get results back from CI.

Reply:
Yep, good catch, I added a `ThermalCameraBoxesHeatSignature` and renamed `ThermalCameraBoxes` to `ThermalCameraBoxesUniformTemperature` in https://github.com/ignitionrobotics/ign-rendering/pull/189/commits/f710f72c018e0caf03a6a77a29881302d4b9263f so that we can distinguish between the two easily